json/src/ser/mod.rs

```diff
@@ -286,6 +284,20 @@ impl<D> Serialize for DicomJson<InMemElement<D>> {
     }
 }
 
+impl Serialize for DicomJson<&PixelFragmentSequence<Vec<u8>>> {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let offset_table = self.inner().offset_table();
+        let fragments = self.inner().fragments();
+        let mut map = serializer.serialize_map(Some(2))?;
+        map.serialize_entry("Offset Table", offset_table)?;
```

Comment:
I have mixed feelings about this. :) It may be interesting for encapsulated pixel data serialization to be supported in some way, but it may also be surprising for this to exist as a non-standard extension provided specifically by this implementation.

I think we'd be able to serve both worlds if we did the following in this order:

1. At dicom-dump, we can check whether Pixel Data is encapsulated, raise a warning that it will be ignored, and remove it from the dataset before the dump.
2. Later on, we can make it possible to provide parameters to the serialization process, so that the decision of whether to serialize encapsulated pixel data or not can be explicitly stated by the API user or the `dicom-dump` user.
   - IIRC we don't quite have a way to do this yet. Intuitively I would make this a field of `DicomJson`, but since it is defined as a single item tuple, adding new fields is likely breaking change (which could probably be mitigated through some tricker, but a breaking change nevertheless).